src/data/future-projects.ts

```diff
@@ -0,0 +1,31 @@
+export interface FutureProject {
+    title: string;
+    description: string;
+    techStack: string[];
+}
+
+export const FUTURE_PROJECTS: FutureProject[] = [
+    {
+        title: 'Courses API',
+        description:
+            'Scrapes course info from the UofA website and provides course data for other projects through an API endpoint.',
+        techStack: ['Python', 'Beautiful Soup', 'Fast API'],
+    },
+    {
```

Comment:
Rename MyScheduler to MyTimetable. We decided MyTimetable sounds better than MyScheduler.